src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Consider whether 'ColorSupplier' instance should be a class member or could be instantiated in the method scope if that's the only place it's used.